lib/modules/manager/tekton/extract.ts

```diff
@@ -96,17 +98,28 @@ function addPipelineAsCodeAnnotations(
     return;
   }
 
+  let countPipelines = 0;
+
   for (const [key, value] of Object.entries(annotations)) {
-    if (!taskAnnotation.test(key)) {
+    let match = annotationRegex.exec(key);
+
+    if (!match) {
       continue;
     }
 
-    const tasks = value
+    if (match?.groups?.type == 'pipeline') {
+      countPipelines++;
+      if (countPipelines > 1) {
+        continue;
+      }
+    }
```

Comment:
why continue here?

Reply:
Wrote here about it: https://github.com/renovatebot/renovate/pull/26753#discussion_r1459198508

So if we wan't to update all pipelines but that's not supported by PAC itself or keep this continue so it only updates the first one which is also what PAC does, but PAC does throw an error if the file is wrongly configured.
How should renovate handle it?

Reply:
if more than one is found then the file is invalid?  then we can simply ignore this case, because we probably never hit this condition at runtime.